.travis.yml

```diff
@@ -8,13 +8,15 @@ env:
   matrix:
   - BUILDTOOL=autotools
   - BUILDTOOL=cmake
-  - BUILDTOOL=autotools_old_compilers
```

Comment:
Commented out is fine. Downloading binaries or binary packages was what I was thinking of indeed. But then I got distracted and started working on something else and didn't come back to it yet.

I figure that if more people report bugs related to old compilers, I'll get motivated again :)